monero-outreach-docs/translations/es/storyboard/refuge_es.md

```diff
@@ -0,0 +1,94 @@
+
+En este primer artículo de una serie de tres, les estaré guiando a través del marco actual de las leyes civiles de decomiso de los EEUU, con varios ejemplos de estas leyes inequitativas y como Monero puede ser usado en condición de refugio para servir mejor a las personas. En la segunda parte, señalaré como el Sudanes de la zona sur puede empoderarse al darse cuenta del poder adquisitivo que brinda Monero en tiempos de colapso macroeconómico y la tercera parte tendrá ejemplos de bancos congelando activos. Esto plantea la amenaza hacia la soberanía del individuo y la solución que ofrece Monero. 
+
+## **Expropiación civil en los EEUU**
+
+*"La idea de ir hacia las personas a través de su propiedad tiene una historia larga. Las teorías son bastante viejas. La práctica prevalece hace muy poco."* - Daniel C. Richman (1999) 
+
+Es simple, [el decomiso civil se refiere al proceso legal](https://www.law.cornell.edu/wex/es/expropriación) mediante el cual los funcionarios encargados "incautan" los activos (dinero, propiedades y otros activos tangibles) de los individuos sospechosos de haber participado en una actividad criminal. *¿Suena noble cierto?* *Después de todo, ¿no sería el mundo un mejor lugar si pudiéramos prevenir algún crimen antes de que ocurriera?* Mientras que esto suena bien en teoría, estos mal definidos e invasivos procesos, desafortunadamente conllevan a incontables problemas y tremendas violaciones por parte del gobierno. 
+
+Partiendo del sigo XVI y con raíces en las leyes marítimas británicas, el decomiso civil se ha vuelto una industria billonaria para el gobierno durante los últimos 30 años. Con un apoyo estatal cada vez mayor, pasaron de una incautación aproximada de $93.7 millones de USD en activos y dinero, a un valor de más de [$2.5 billones de USD para el 2010](https://www.wsj.com/news/articles/SB10001424053111903480904576512253265073870). 
+
+Y mientras pueda que exista mérito, causa probable o preponderancia de la evidencia a favor de la incautación de algunos activos (dentro de EEUU los estados difieren frente a las normas sobre las pruebas de las leyes de decomiso), los cálculos a través de los años, [estiman que un 85%](http://www.sfgate.com/news/nation-world/article/Asset-forfeiture-both-an-effective-tool-4546043.php) de las instancias de decomiso civil realizadas son injustificables y superficiales. En los EEUU, tan solo en el 2010, se estima que hubo un promedio de 11,000 decomisos de casos no criminales, de los cuales solamente una décima parte de ellos fueron alegados en cortes federales. Un cálculo realizado por un escritor de SFgate, Stewart M. Powell, indica que el retorno de los decomisos federales a sus dueños se [aproximan a un escaso 1%](https://www.sfgate.com/news/nation-world/article/Asset-forfeiture-both-an-effective-tool-4546043.php). 

```

Comment:
I'd like to suggest this, again this is not wrong but I think it can be more understandable this way:

+ ... (dentro de _**los**_ EEUU _**sus**_ estados difieren _**en su respectivo estándar**_ frente a las normas sobre las pruebas de las leyes de decomiso)

Reply:
Great, better structured. Changed*